lib/elixir/lib/module/locals_tracker.ex

```diff
@@ -213,15 +213,24 @@ defmodule Module.LocalsTracker do
 
   defp get_line(meta), do: Keyword.get(meta, :line)
 
+  defp get_position(meta), do: maybe_add_col([line: get_line(meta)], meta)
```

Comment:
Because this position is passed around between processes, we want to keep it light. So one suggestion is to have it either as an integer `line` or as a tuple`{line, column}`. Then `build_meta` unpacks that into keyword lists.